charts/cyclecloud/values.yaml

```diff
@@ -44,6 +44,7 @@ cycle:
   forceDeleteVMSSsEnabled : false
   useWorkloadIdentity: false
   storage_managed_identity: "<resource-id>"
+  not_generate_cs_config: "false"
```

Comment:
is there a reason for the inverted value here?   
Unless we need "non_gener.." for backwards compat support I'd prefer 
generate_cs_config : "true"

Reply:
ok will change to generate_cs_config